tap/tlstapper/chunk.go

```diff
@@ -12,8 +12,8 @@ import (
 const FLAGS_IS_CLIENT_BIT uint32 = (1 << 0)
 const FLAGS_IS_READ_BIT uint32 = (1 << 1)
 
-func (c *tlsTapperTlsChunk) getAddress() (net.IP, uint16, error) {
-	address := bytes.NewReader(c.Address[:])
+func (c *tlsTapperTlsChunk) getFdAddress() (net.IP, uint16, error) {
```

Comment:
Let's get rid of this

Reply:
Removed cf6598b8e5f6eecb568c7566122cba75cdf8d3db.